arm-network/2017-06-01/swagger/networkSecurityGroup.json

```diff
@@ -468,9 +469,23 @@
           "type": "string",
           "description": "The CIDR or source IP range. Asterix '*' can also be used to match all source IPs. Default tags such as 'VirtualNetwork', 'AzureLoadBalancer' and 'Internet' can also be used. If this is an ingress rule, specifies where network traffic originates from. "
         },
+        "sourceAddressPrefixes": {
```

Comment:
@ravbhatnagar Is adding new non-readonly fields allowed on the response ? I think existing SDKs would ignore these fields if sent by the server. Could you please throw some light here. Thanks!